app.rb

```diff
@@ -0,0 +1,136 @@
+# require_relative 'lib/database_connection'
+# require_relative 'lib/artist_repository'
+# require_relative 'lib/album_repository'
+
+# DatabaseConnection.connect('music_library')
+
+# artist_repository = ArtistRepository.new
+# album_repository = AlbumRepository.new
+
+# artist_repository.all.each do |artist|
+#  p artist
+# end
+
+# album_repository.all.each do |album|
+#   puts "#{album.id} - #{album.title} - #{album.release_year}"
+#  end
+
+# # result = DatabaseConnection.exec_params('SELECT * FROM artists;', [])
+
+# # result.each do |record|
+# #     p record
+# # end
+
+# artist = artist_repository.find(4)
+# puts artist.name
+
+# album = album_repository.find(4)
+# puts album.title
+
```

Comment:
It's good to see that you've referred to previous projects to help with this, however i would remove them before committing just to save the extra lines